core/src/main/java/org/polypheny/db/algebra/enumerable/lpg/EnumerableLpgMatch.java

```diff
@@ -147,7 +147,10 @@ private Expression getPolyElement( RexCall call ) {
 
     @Override
     public String algCompareString() {
-        return "$" + getClass().getSimpleName() + "$" + matches.hashCode() + "$" + names.hashCode();
+        return getClass().getSimpleName() + "$"
+                + matches.hashCode() + "$"
+                + names.hashCode() + "$"
```

Comment:
Is the hash code stable enough for the purposes of this compare sting?

Reply:
Removed as it is already covered by parent.